src/js/components/search/collapsibleSidebar/SearchSidebarDrilldown.jsx

```diff
@@ -30,6 +30,30 @@ const SearchSidebarDrilldown = ({
         }
     };
 
+    let categoryFilter;
+
+    if (selectedCategory?.title === 'Location' || selectedCategory?.title === 'Time Period') {
```

Comment:
i think the category filter should always just be the header without the icon and description.  we could double check with marcy because the mocks are inconsistent.  pending Marcy's response we could probably remove the -36 margin top from the categoryfilter component

Reply:
ok based on marcy's comment, we should pass a level prop to the categoryfilter to determine if the icon and description show.  if level 3 only show the text.  and if level 3 don't add the margin-top -36px style

Reply:
done!